src/Connection/Replication/SentinelReplication.php

```diff
@@ -315,31 +316,30 @@ public function getSentinelConnection()
      */
     public function updateSentinels()
     {
-        SENTINEL_QUERY: {
+        SENTINEL_QUERY:
             $sentinel = $this->getSentinelConnection();
 
-            try {
-                $payload = $sentinel->executeCommand(
-                    RawCommand::create('SENTINEL', 'sentinels', $this->service)
-                );
-
-                $this->sentinels = [];
-                $this->sentinelIndex = 0;
-                // NOTE: sentinel server does not return itself, so we add it back.
-                $this->sentinels[] = $sentinel->getParameters()->toArray();
-
-                foreach ($payload as $sentinel) {
-                    $this->sentinels[] = [
-                        'host' => $sentinel[3],
-                        'port' => $sentinel[5],
-                        'role' => 'sentinel',
-                    ];
-                }
-            } catch (ConnectionException $exception) {
-                $this->sentinelConnection = null;
-
-                goto SENTINEL_QUERY;
+        try {
+            $payload = $sentinel->executeCommand(
+                RawCommand::create('SENTINEL', 'sentinels', $this->service)
```

Comment:
Let's remove this code changes please.

Reply:
@tillkruss This changes related to phpcsfixer library update that used as a linter and step in CI pipeline, this changes how nothing code wise it's all about indents. If I remove this changes CI will fail

Reply:
That was disabled: https://github.com/predis/predis/commit/a9d18b182ca1efe43ad3066520fe1bf566b5c189

Reply:
Ah, perfect! Changes was reverted